### DISCUSSION TITLE
Installation on different folder?

### DISCUSSION THREAD

**Discussion by nikidubois** [ubergeek77/Lemmy-Easy-Deploy#59] [General]
Hi,

I'm running Lemmy-easy-deploy on Hetzner servers.
I am not super into running servers etc. I installed LED in the root of the server.
I'm now noticing that I'm starting to run out of disk space.

I can buy extra space at Hetzner. The extra space will be a new mount. I think I must move everything to there or just some folders?
Is this something that can be done?

**Comment by ubergeek77** [comment]
Just to make sure you know - the `Lemmy-Easy-Deploy` folder, and the `live` folder inside of it, store basically zero data. They're just configuration files and passwords.

Since I use Docker Volumes, the real data is stored inside the Docker system directory, and unfortunately I can't re-mount that for you.

However, this was discussed previously, and I have a link to a guide where you can change the path of your full Docker system directory to the new mount:

https://github.com/ubergeek77/Lemmy-Easy-Deploy/issues/47

Just make sure to keep a safe backup of the `live` folder, as important passwords/secrets are stored there!

Once you've done all this, you don't even need to move the `Lemmy-Easy-Deploy` folder. You can just keep it where it is, and as long as you have configured the Docker Daemon to store its data on a different mount point, then you should be good to go.

As a disclaimer, I've never done this process, so I'm not sure how it would work for existing volumes, but I believe that guide walks you through copying everything.

But remember my golden rule on doing migrations across disks - Always *copy*, never *move!* 😅

**Reply by nikidubois** [reply]
It worked! I used this guide: https://stackoverflow.com/questions/59345566/move-docker-volume-to-different-partition
